src/components/TreeView/stories/treeViewManagerStory.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import template from './treeViewStoryDescriptionTemplate';
+
+import MockTreeViewManager from '../__mocks__/MockTreeViewManager';
+
+const description = (
+  <div>
+    <h3>Notes</h3>
+    <ul>
+      <li>
+        Add Mappings simulates a scenario where a user has set up two mappings
+      </li>
+      <li>
+        <a href="https://github.com/patternfly/patternfly/blob/master/src/sass/converted/patternfly/_bootstrap-treeview.scss#L74-L88">
+          Styles for selected and hover states
+        </a>{' '}
+        may need adjustment in order to control highlighting and cursor of
+        unselectable elements
+      </li>
+    </ul>
+  </div>
+);
+
+const treeViewManagerStory = stories => {
+  stories.addWithInfo('Mock TreeView Manager', '', () => {
+    const story = <MockTreeViewManager />;
+
+    return template('Mock TreeView Manager', story, description);
+  });
+};
```

Comment:
The storybook would be much more helpful if it included the code necessary to create this mock. Currently, there isn't much there that is of any help.

Reply:
@jeff-phillips-18 @priley86 I created this mock mainly as an integration test for the v2v wizard. Is it appropriate to include this in the storybook, or should I remove it?

I've fixed up the main example with knobs, source, and prop tables.

I'm happy to add more as well, just not sure if we want to keep this particular one